esmero/strawberryfield
# ISSUE-53: Create new Config page for Solr settings

Addresses #53 

### Issue Summary
This push addresses the first part only of #53, creating the form to save the new config.

### What's changed?
- Add schema for strawberryfield.archipelago_solr_settings. For now it only holds type related config
- Add route to /admin/config/archipelago/important-solr-settings
- Add ImportantSolrSettingsForm.php
  This is the most code in this file. I don't know if it is the optimal UI, with multiple dropdowns, but it works pretty well for me.

### To Test
Go to /admin/config/archipelago/important-solr-settings and follow the form
I created extra Servers and Indexes to make sure all the conditional switching worked.

### Notes
Not sure how you want to deal with no config set. I am guessing, @DiegoPino since you were going to deal with the second part of #53 , that you might set default config.
